File: 09_week_1/pushFront.js
/*	pushFront	*/
function pushFront(arr, val) {

	for (var j = arr.length; j > 0; j--) {
		arr[j] = arr[j-1];
	}
	arr[0] = val;
}

var arr = [3,4,"hello", [1,2, "hi"]];
pushFront(arr, "new_value");
console.log(arr);

File: 09_week_1/pushFront.py
# pushFront
def pushFront(arr, val):

	arr.append(0)
	for idx in range(len(arr)-1, 0, -1):
		arr[idx] = arr[idx-1]
	arr[0] = val


arr = [1,"sweet", 65]
pushFront(arr, 5)
print(arr)
